tools/run_tests/xds_k8s_test_driver/bin/run_td_setup.py

```diff
@@ -151,6 +154,24 @@ def main(argv):
                                          tls=True,
                                          mtls=False)
 
+            elif security_mode == 'server_authz_error':
+                # Error case: client does not authorize server
+                # because of mismatched SAN name.
+                logger.info('Setting up mtls_error')
+                td.setup_for_grpc(server_xds_host, server_xds_port)
+                # Regular TLS setup, but with client policy configured using
```

Comment:
This code is identical to the one in security_test.py:155-166. How about moving the logic to `TrafficDirectorManager` which sets up the client and server security for each test-case? Just a thought.

Reply:
This is a helper script, I wanted to keep it explicit. But I agree, It's probably a good time to move some of this code to helpers. Will make this change as a separate PR to not block delivering this test.